tests/TestCase/ORM/EntityTest.php

```diff
@@ -1591,6 +1591,16 @@ public function testGetSetInvalid()
             'title' => 'albert',
             'body' => 'einstein'
         ], $entity->getInvalid());
+
+        $overwrite = $entity->setInvalid([
+            'title' => 'nikola',
+            'body' => 'tesla'
+        ], true);
```

Comment:
Yeah, but we need a test that it works as expected ie does not overwrite current state 😀